From 21faba6831fa02eadf72e81bd78f8c5969d26efb Mon Sep 17 00:00:00 2001
From: mikramarc <michal@shadowrobot.com>
Date: Mon, 2 Aug 2021 15:14:58 +0100
Subject: [PATCH] Create postinst

---
 debian/postinst | 14 ++++++++++++++
 1 file changed, 14 insertions(+)
 create mode 100644 debian/postinst

diff --git a/debian/postinst b/debian/postinst
new file mode 100644
index 0000000..c760128
--- /dev/null
+++ b/debian/postinst
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
+ROS_DISTRO=noetic
+EXEC_NAME=sr_hand_detector_node
+case "$1" in
+ configure)
+   setcap cap_net_raw+ep /opt/ros/$ROS_DISTRO/bin/$EXEC_NAME
+   ;;
+ *)
+       echo "postinst called with unknown argument \`$1'" >&2
+       exit 0
+   ;;
+esac
+#DEBHELPER#
-- 
2.25.1

